docs/DeveloperGuide.md

```diff
@@ -51,7 +51,7 @@ For example, the `Logic` component (see the class diagram given below) defines i
 
 **How the architecture components interact with each other**
 
-The *Sequence Diagram* below shows how the components interact with each other for the scenario where the user issues the command `delete 1`.
+The *Sequence Diagram* below shows how the components interact with each other for the scenario where the user issues the command `delete A1234567X`.
```

Comment:
Would "In the scenario" be better than "for the scenario"?

Reply:
Resolved!